src/Kestrel.Core/Internal/Http2/Http2OutputProducer.cs

```diff
@@ -25,7 +26,7 @@ public void Dispose()
         {
         }
 
-        public void Abort(Exception error)
+        public void Abort(ConnectionAbortedException error)
```

Comment:
ConnectionAbortedException may not make sense. You're aborting the request stream.

Reply:
I don't think I understand what you're trying to get at here. Since writing to the response stream no longer fails absent a CancellationToken, the ConnectionAbortedException is only observed via the RequestStream in HTTP/1 as well.

Reply:
I'm describing the nature of aborting in Http2 and passing in a ConnectionAbortedException for a request abort when it has nothing to do with aborting the underlying "connection". It's just semantics but it's not a hill I would die on as part of this PR.